lib/pages/image_viewer/image_viewer.dart

```diff
@@ -32,13 +36,47 @@ class ImageViewerController extends State<ImageViewer> {
 
   final ValueNotifier<bool> showAppbarPreview = ValueNotifier(true);
 
+  String? filePath;
+
+  final downloadMediaFileInteractor = getIt.get<DownloadMediaFileInteractor>();
+
+  StreamSubscription? streamSubcription;
+
   @override
   void initState() {
     super.initState();
+    if (!PlatformInfos.isWeb && widget.event != null) {
```

Comment:
why not in web?

Reply:
this is file for native only, i will do same logic for web

Reply:
i think i will create another ticket for improve web

Reply:
https://github.com/linagora/twake-on-matrix/issues/1704